dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/custom/package.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
I believe all we need to do is to add doc for the customer to create the appropriate package instead of adding it OOTB

Reply:
Sure probably we need to docu in the website some where. This is just to avoid an error when looking for in this package when there is nothing

Reply:
what is the intention of this page? Is it like a readme for developers?

Reply:
the idea of the branch is to scan for classes annotated by in the java packages, however it needs a start point on plugins, this package is for it